src/elements/inputTags/InputTags.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-unused-expressions */
+import React, { FC, useRef } from 'react';
+import classnames from 'classnames';
+import InputLabel from '../inputLabel';
+import Tag from './Tag';
+import Props from './InputTags.types';
+
+const InputTags: FC<Props> = ({
+  tags = [],
+  name,
+  id,
+  className,
+  dataTestId,
+  placeholder,
+  helperText,
+  disabled,
+  seprators = ['Enter', 'Comma'],
+  maxTags = 8,
+  minLength = 1,
+  maxLength = 30,
+  size = 'lg',
+  inputLabelProps,
+  onChange,
+  onRemoved,
+  onExisting,
+  onBlur,
+}: Props): JSX.Element => {
+  const ref = useRef(null);
+
+  const handleTagRemove = text => {
+    onChange && onChange(tags.filter(tag => tag !== text));
+    onRemoved && onRemoved(text);
+    const textInput = ref.current;
+    textInput.focus();
+  };
+
+  const handleOnKeyUp = e => {
+    e.stopPropagation();
+    const text = e.target.value.trim();
+
+    if (
+      text.length >= minLength &&
+      text.length <= maxLength &&
+      tags.length < maxTags &&
+      (seprators.includes(e.key) || seprators.includes(e.code))
+    ) {
+      if (tags.includes(text)) {
+        onExisting && onExisting(text);
+        return;
+      }
+      onChange && onChange([...tags, text]);
+      e.target.value = '';
+      e.preventDefault();
+    }
+  };
+
+  return (
+    <div className={classnames(`size-${size}`, 'form-group')}>
```

Comment:
The prop "className" that the component receives is usually passed to the first div of the component, so we can customize the style of the entire component and not only the style of the input. Can you move it up here?

Reply:
Yes, you are right. Done 👍